core/src/main/resources/jenkins/model/Jenkins/manage.jelly

```diff
@@ -60,10 +60,8 @@ THE SOFTWARE.
   <l:main-panel>
     <h1>${%Manage Jenkins}</h1>
 
-    <j:forEach var="am" items="${app.administrativeMonitors}">
-      <j:if test="${am.isActivated() and am.isEnabled()}">
```

Comment:
Yep. Initially I thought of simply doing that but given that I was going to create an issue, a PR and so on, I thought it was convenient to add the (still small) additional changes:
* [x] Use the same logic, from the same place in the manage Jenkins page and the decorator (the decorator will simply forward to `Jenkins` once the TODOs are solved).
* [x] Remove that logic from the Jelly.